### PR TITLE
Feature/271 add manual stap capability

### PR DESCRIPTION
**Pull request by MaartendeKruijf**

### CODE REVIEW

File: pkg/core/capability/manual/manual.go
Comment (lucamrgs) [inline pkg/core/capability/manual/manual.go:46]: Context has all data we need to execute a command, right? Should we rename "context" to e.g. "commandContext"?  Just using "context" is at least to me confusing to interpret right away.

File: pkg/interaction/interaction.go
Comment (lucamrgs) [inline pkg/interaction/interaction.go:1]: I feel like this file should go in pkg/core/capability/manual/ as this is really just related to manual execution, no?

Comment (MaartendeKruijf) [reply pkg/interaction/interaction.go:1]: I will merge it with your package as soon as its merged, we can leave the PR open till then.

**Review by RabbITCybErSeC — APPROVED** [review]

File: pkg/core/capability/manual/manual.go
Comment (lucamrgs) [inline pkg/core/capability/manual/manual.go:26]: Why is the channel defined by argument, and not automatically upon creation?

Comment (lucamrgs) [reply pkg/core/capability/manual/manual.go:26]: And also why does not this return a pointer to the capability?

File: pkg/core/capability/manual/manual.go
Comment (lucamrgs) [inline pkg/core/capability/manual/manual.go:28]: Can we discuss on whether we need one channel per whole playbook execution, or one channel per each manual command execution?

As you implemented it, is one per playbook execution, but I was thinking we might need one per command execution.

This is because if we have a parallel step with two manual commands, assuming parallel steps is actually asynchronous, then if we have only one manual capability per playbook execution, we might get the wrong response in a parallel branch, if the response from the other parallel branch was executed first, right?

**Review by lucamrgs — CHANGES_REQUESTED** [review]

File: pkg/core/capability/manual/manual.go
Comment (lucamrgs) [reply pkg/core/capability/manual/manual.go:63]: @MaartendeKruijf by what I am implementing, I think using the context would be better also because we can also pass it to the Queue(function), capture the timeout inside the interaction object, and thus handle the idle-wait on the go routines Notify for integrations response more gracefully. Right?

**Review by lucamrgs — CHANGES_REQUESTED** [review]

**Pull request merged**
